doc/ufunc_examples.rst

```diff
@@ -0,0 +1,317 @@
+.. _ufunc_examples:
+
+Ufunc Examples
+--------------
+
+This page contains examples of different Grid Ufuncs that you might find useful.
+They are intended to be more advanced or realistic cases than the simple differencing operations we showed in the introduction to :ref:`grid_ufuncs`.
+
+As each of these cases is a vector calculus operator, we will demonstrate their use on an example two-dimensional vector field.
+
+Firstly we need a two-dimensional grid, and we use similar coordinate names to the 1D example we used when first introducing :ref:`grids`.
+
+.. ipython:: python
+
+    import numpy as np
+    import xarray as xr
+
+    grid_ds = xr.Dataset(
+        coords={
+            "x_c": (
+                [
+                    "x_c",
```

Comment:
I am very convinced, that when you remove the trailing comma after `x_c`, black is going to format this inline as 
```
["x_c"]
```
which would make those code blocks a **lot** easier to read.

Reply:
That worked! I applied it across other pages too.